third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -1277,37 +1277,45 @@ const ParagraphStyle& ParagraphTxt::GetParagraphStyle() const {
 }
 
 double ParagraphTxt::GetAlphabeticBaseline() {
+  FML_DCHECK(!needs_layout_) << "only valid after layout";
```

Comment:
I don't think these will fire on test runs for the framework - right?

Reply:
Does framework run any debug build tests? If not, should it? There are far more asserts in debug mode than the others and would catch more issues.

Reply:
Since these checks were not enforced before at this level, I was hesitant to add regular FML_CHECK as i feared it may cause apps to unexpectedly assert-crash.

Reply:
I think I'll land this first, and see if we can get a host_debug suite of test runs on framework.